README.md

```diff
@@ -424,31 +366,6 @@ To initiate the test suites, run:
 npm run test
 ```
 
-### Code Style
```

Comment:
We have to keep the 'Code Style', we just removed the last paragraph about `Husky`.

Reply:
Done